lib/app/app.dart

```diff
@@ -99,6 +100,9 @@ class _AppState extends State<App> {
             create: (BuildContext context) =>
                 ProfilesCubit(getIt(), getIt(), getIt(), getIt(), getIt()),
           ),
+          BlocProvider(
```

Comment:
I remember us discussing in the arch meeting that we preferred injecting the cubits via getit rather than relying on them to be passed via context.